system-addon/content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -93,7 +96,23 @@ class CollapsibleSection extends React.PureComponent {
     this.onInfoLeave = this.onInfoLeave.bind(this);
     this.onHeaderClick = this.onHeaderClick.bind(this);
     this.onTransitionEnd = this.onTransitionEnd.bind(this);
-    this.state = {isAnimating: false, infoActive: false};
+    this.enableOrDisableAnimation = this.enableOrDisableAnimation.bind(this);
+    this.state = {enableAnimation: false, isAnimating: false, infoActive: false};
+  }
+
+  componentWillMount() {
+    this.enableOrDisableAnimation();
```

Comment:
Do you need this initial call to ```enableOrDisableAnimation()```? I removed it and it seemed to work the same since the function is called anyways when you open a new tab because of the event listener firing

Reply:
I guess to be safe that all visible tabs start with ```animation-enabled``` you could set ```enableAnimation : true``` in the constructor

Reply:
I *think* we need to call `enableOrDisableAnimation()` on mount in case it is initially a non-visilble tab that is loading. Or does that still trigger a "visibilitychange" event? I'll play with it a little.

Reply:
Actually, you're right. It doesn't seem to be needed 👍